### PR TITLE
Various vox changes for preview

### PR DESCRIPTION
**Pull request by Rayfod**

### CODE REVIEW

**Review by Anton-Kr — CHANGES_REQUESTED** [review]
Make a separate chaplain locker

File: maps/away/voxship/voxship_jobs.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 /datum/job/submap/voxship_vox/doc
 	title = "Shard Biotechnician"
-	total_positions = 1
+	total_positions = 2
```
Comment (Anton-Kr) [inline maps/away/voxship/voxship_jobs.dm]: Reduce other vox roles to make total be the same.

File: maps/away/voxship/voxship_jobs.dm [hunk 2]
```diff
@@ -202,17 +202,14 @@
...
-/obj/item/weapon/gun/energy/plasmastun/vox
-	desc = "The modified Hephaestus Industries MA21 Selkie is a weapon that uses a laser pulse to ionise the local atmosphere, creating a disorienting pulse of plasma and deafening shockwave as the wave expands. Without a local atmosphere to ionize, however, it becomes a very expensive paperweight. This model seems heavily modified, to use the power of biofuel."
-	self_recharge = 1
-	recharge_time = 20
 
 /obj/item/weapon/gun/energy/sniperrifle/vox
 	desc = "This is a modified Hephaestus Industries Baleful. The cell has been replaced by a vox variant, allowing it to self-charge. Sadly, this makes it lose its function to knock people out. It's a designated marksman rifle capable of shooting powerful ionized beams."
 	self_recharge = 1
-	recharge_time = 80
+	recharge_time = 60
```
Comment (Anton-Kr) [inline maps/away/voxship/voxship_jobs.dm]: no thanks

File: maps/away/voxship/voxship_jobs.dm [hunk 3]
```diff
@@ -187,12 +187,12 @@
...
-	options["Stolen Marine Gear"] = list(/obj/item/weapon/gun/projectile/automatic/bullpup_rifle,/obj/item/weapon/storage/firstaid/adv,/obj/item/ammo_magazine/mil_rifle,/obj/item/ammo_magazine/mil_rifle,/obj/item/ammo_magazine/mil_rifle,/obj/item/weapon/gun/energy/gun,/obj/item/weapon/storage/box/fragshells)
+	options["Medic gear"] = list(/obj/item/weapon/gun/energy/darkmatter,/obj/item/weapon/gun/launcher/alien/slugsling,/obj/item/weapon/storage/firstaid/combat,/obj/item/clothing/glasses/hud/health/visor,/obj/item/device/scanner/health,/obj/item/weapon/storage/firstaid/adv)
+	options["Enforcer"] = list(/obj/item/weapon/gun/energy/darkmatter,/obj/item/weapon/gun/launcher/alien/spikethrower,/obj/item/weapon/storage/firstaid/adv,/obj/item/weapon/gun/projectile/automatic/machine_pistol,/obj/item/ammo_magazine/machine_pistol,/obj/item/ammo_magazine/machine_pistol,/obj/item/ammo_magazine/machine_pistol,/obj/item/weapon/gun/projectile/automatic/machine_pistol,/obj/item/ammo_magazine/machine_pistol,/obj/item/ammo_magazine/machine_pistol,/obj/item/ammo_magazine/machine_pistol)
```
Comment (Anton-Kr) [inline maps/away/voxship/voxship_jobs.dm]: One machine pistol is sufficient here

**Review by Anton-Kr — CHANGES_REQUESTED** [review]

**Review by Anton-Kr — CHANGES_REQUESTED** [review]
Fix CI issues.

**Comment by Rayfod** [comment]
Going to wait a bit to get the main vox changes out of the way

**Pull request closed without merging**